WalletWasabi.Tests/UnitTests/Stores/IndexStoreTests.cs

```diff
@@ -35,141 +32,6 @@ public async Task IndexStoreTestsAsync()
 		await indexStore.InitializeAsync(testDeadlineCts.Token);
 	}
 
-	[Fact]
-	public async Task InconsistentMatureIndexAsync()
```

Comment:
i agree. Corrupting SQLite is more difficult, i know that because the heart of NixOS is built on top of SQLite and it never failed to me even when I suffered power outages during system upgrades. I don't mean it is not corruptible, just pretty robust in that matter.

Reply:
Out of curiosity, are you using NixOS as a desktop general purpose OS?